src/main/java/com/minecolonies/core/entity/ai/workers/util/ConstructionTapeHelper.java

```diff
@@ -43,13 +46,11 @@ private ConstructionTapeHelper() {}
      */
     public static void placeConstructionTape(@NotNull final IWorkOrder workOrder, @NotNull final Level world, final IColony colony)
```

Comment:
Can't we do it async?

Reply:
It's already calculated, there's no point calculating it again.  Plus the act of calculating it re-rotated the blueprint, since `loadBlueprint` returns a (possibly cached) pre-rotated copy.

Reply:
ah, fair

Reply:
It's probably bad coding style that `calculateCorners` (not this one, but the call that was removed) also mutates the blueprint, but that's probably a refactor for another time.